File: javascript/src/training/easy/onboarding.js
while (true) {
    const enemy1 = readline(); // name of enemy 1
    const dist1 = parseInt(readline()); // distance to enemy 1
    const enemy2 = readline(); // name of enemy 2
    const dist2 = parseInt(readline()); // distance to enemy 2

    // Write an action using console.log()
    let target = enemy1;
    if (dist1 > dist2) {
        target = enemy2;
    }

    // You have to output a correct ship name to shoot ("Buzz", enemy1, enemy2, ...)
    console.log(target);
}

File: python/src/training/easy/onboarding.py
import sys


def main():
    while True:
        enemy1 = input()  # name of enemy 1
        dist1 = int(input())  # distance to enemy 1
        enemy2 = input()  # name of enemy 2
        dist2 = int(input())  # distance to enemy 2

        target = enemy1
        if dist1 > dist2:
            target = enemy2

        # You have to output a correct ship name to shoot ("Buzz", enemy1, enemy2, ...)
        print(target)


if __name__ == '__main__':
    main()
